assets/csidriveroperators/shared-resource/09_deployment.yaml

```diff
@@ -17,6 +17,18 @@ spec:
         name: shared-resource-csi-driver-operator
     spec:
       containers:
+      - image: ${WEBHOOK_IMAGE}
```

Comment:
Typically the webhook would be in its own deployment @jkhelil, not a separate container in the operator's deployment.

I believe I missed planning for this.  Were you given direction there by @coreydaley or @adambkaplan to do it this way?

I would be surprised if that was the case.  If so, let's discuss.  If not though, adjust this PR to create a separate deployment.

Reply:
+1 - the webhook should be in a separate deployment.

Reply:
so yeah additionally besides the separate commit remove this file from this PR and include it in a shared resource operator PR per discussion in slack